src/interfaces/keystore.ts

```diff
@@ -116,6 +119,8 @@ export type ExternalKey = {
     deviceModel: string
     hdPathTemplate: HD_PATH_TEMPLATE_TYPE
     index: number
+    timestamp: number
```

Comment:
I'd call this `createdAt` (we use `createdAt` and `updatedAt` for the Address book, inspired by the NodeJS conventions).